spec/test_spec.rb

```diff
@@ -1 +1,94 @@
+require 'fileutils'
+require 'pry'
+require 'rake'
 
+RSpec.describe "1" do
```

Comment:
Start using meaning full names instead of "1"

Reply:
done